File: Python/Parite2/Exo2.py
import os

pid_fils1 = os.fork()
if pid_fils1 == 0:
    print("je suis le fils 1")
else:
    pid_fils2 = os.fork()
    if pid_fils2 == 0:
        print("je suis le fils 2")
    else:
        print("je suis le père")
        os.wait()
        os.wait()

File: C/Parite2/Exo2.c
#include <stdlib.h>
#include <stdio.h>
#include <unistd.h>
#include <sys/wait.h>
#include<time.h>

int main()
{
    pid_t pid1, pid2;

    pid1 = fork();
    if(pid1 == 0)
    {
        printf("Je suis le fils 1");
        return 0;
    }
    pid2 = fork();
    if(pid2 == 0)
    {
        printf("\nJe suis le fils 2");
    }
    else if(pid2 > 0)
    {
        printf("\nJe suis le pere");
    }
    else
    {
        printf("\nErreur");
    }
    return 0;
}